developing-with-nats/events/README.md

```diff
@@ -25,8 +25,6 @@ getStatusTxt := func(nc *nats.Conn) string {
 }
 log.Printf("The connection is %v\n", getStatusTxt(nc))
 
-nc.Close()
```

Comment:
The connection is closed by defer statement, but we explicitly close here to show the difference of the status. See how we print the status before and after the close to show the difference.

Reply:
yeah that makes sense